core/src/main/java/io/kestra/core/models/executions/AbstractMetricEntry.java

```diff
@@ -23,7 +24,7 @@
     @JsonSubTypes.Type(value = Timer.class, name = "timer"),
 })
 @ToString
-@EqualsAndHashCode
+@EqualsAndHashCode(exclude="creationTime")
```

Comment:
Seems to be too magical, maybe have a proper method compare instead of really on `equals`?

Reply:
Isn't it the purpose of Lombok to provide magical highly productive trick ?